pages/profile/permission/content/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  fields: require('./fields'),
+  buttons: {
+    submit: 'Save',
+    remove: 'Remove'
+  },
+  title: 'Change permission level',
+  remove: 'Remove permissions for this establishment',
+  copy: 'If you remove the user from this establishment they will no longer be able to view any of this establishment\'s data and all currently held licenses will need to either be transferred to another establishment or put on hold'
```

Comment:
I'm not sure this content corresponds with the technical reality. It's probably more a question for @pcuk1979 but I think we need them to remove/revoke associated roles/licences _before_ being able to remove the user.

Reply:
To confirm : do we need to amend the copy ?

Reply:
This is ticket https://jira.digital.homeoffice.gov.uk/browse/AS-834.